### PR TITLE
feat: :sparkles: init GraphCMS w/ Astro

### PR DESCRIPTION
**Pull request by spences10**
Still to add a readme

### CODE REVIEW

File: with-astro/.npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## force pnpm to hoist 
+shamefully-hoist = true
```
Comment (jpedroschmitz) [inline with-astro/.npmrc:2]: I think this file can be removed. We're not using pnpm here, right?

File: with-astro/.stackblitzrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (jpedroschmitz) [inline with-astro/.stackblitzrc:1]: This file can also be deleted.

File: with-astro/src/pages/index.astro [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        {
+          products {
+            slug
+            name
+          }
+        }
+      `,
+    }),
+  }
+)
```
Comment (jpedroschmitz) [inline with-astro/src/pages/index.astro:23]: Maybe we could wrap this fetch in a separate file, and import it here and in the other files that has the `fetch`.

**Review by jpedroschmitz — COMMENTED** [review]
I noticed two files that can be deleted. Other then that, everything looks great ❤️ 

Awesome job, @spences10 🚀

File: with-astro/.npmrc (hunk 1 above)
Comment (spences10) [reply with-astro/.npmrc:2]: Done

File: with-astro/.stackblitzrc (hunk 2 above)
Comment (spences10) [reply with-astro/.stackblitzrc:1]: Done

File: with-astro/src/pages/index.astro (hunk 3 above)
Comment (spences10) [reply with-astro/src/pages/index.astro:23]: Think this will be fine as is, simple example to demonstrate how to use it

**Comment by notrab** [comment]
Looks good @spences10 

Can we add a link to this on the root project README, and merge?

**Comment by spences10** [comment]
Ah! Yeah, that would be the other thing! 🤦

Looks like I'll need to do that for eleventy too!

I'll get them added then merge this if @notrab and @jpedroschmitz are ok with that?

**Comment by spences10** [comment]
That's done now

**Comment by notrab** [comment]
Awesome. Looks great

**Pull request merged**